Makefile

```
@@ -1,4 +1,4 @@
-DOCKER := $(shell which docker)
+pOCKER := $(shell which docker)
```

I think this is a typo

```suggestion
DOCKER := $(shell which docker)
```